README.md

```diff
@@ -1 +1,32 @@
-# eversim
\ No newline at end of file
+# eversim
+
+eversim is a 2D game engine written in modern C++. Its goal is to support
+games which depend on different systems to simulate a living and breathing world
+and to create emergent gameplay. See the videos of GMTK about [immersive
+sims](immersive sim) and [systemic games](systemic game) for reference.
+
+[immersive sim]: https://www.youtube.com/watch?v=kbyTOAlhRHk
+[systemic game]: https://www.youtube.com/watch?v=SnpAAX9CkIc
+
+It runs on Windows and support for Linux is in work.
```

Comment:
Currently it compiles, runs and has the same visual bugs on both, so actually is does already run on both :)

Reply:
Current master doesn't build on linux yet.